generators/languages/templates/src/main/webapp/i18n/de/register.json

```diff
@@ -10,14 +10,14 @@
                     "required": "Ihr Benutzername wird benötigt.",
                     "minlength": "Ihr Benutzername muss mindestens 1 Zeichen lang sein",
                     "maxlength": "Ihr Benutzername darf nicht länger als 50 Zeichen sein",
-                    "pattern": "Your username is invalid."
+                    "pattern": "Ihr Benutzername ist ungültig."
                 }
             },
-            "success": "<strong>Registrierung gespeichert!</strong> Bitte überprüfen Sie Ihre Emails für die Bestätigung.",
+            "success": "<strong>Registrierung gespeichert!</strong> Bitte überprüfen Sie Ihre E-Mails für die Bestätigung.",
             "error": {
                 "fail": "<strong>Registrierung fehlgeschlagen!</strong> Bitte versuchen Sie es später nochmal.",
                 "userexists": "<strong>Benutzername bereits vergeben!</strong> Bitte wählen Sie einen anderen aus.",
-                "emailexists": "<strong>Email wird bereits verwendet!</strong> Bitte wählen Sie eine andere aus."
+                "emailexists": "<strong>E-Mailadresse wird bereits verwendet!</strong> Bitte wählen Sie eine andere aus."
```

Comment:
missing space ?

Reply:
You mean `E-Mailadresse`? I think it should be `E-Mail-Adresse` according to the [Duden](https://www.duden.de/rechtschreibung/E_Mail_Adresse)

Reply:
yes, you know better than me :-)

Reply:
In fact I also had to look it up :)

Reply:
@atomfrede fixed